grammars/python.cson

```diff
@@ -1151,7 +1151,7 @@
         'name': 'string.quoted.double.block.sql.python'
         'patterns': [
           {
-             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER))'
+             'begin': '(?=\\s*(SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER|WITH))'
```

Comment:
I'm not sure I understand where the forth option you list needs additional coverage. I found all the areas where quoted strings start with SQL keywords, and I can find not further instances.